devenv/setupUbuntuOnPPC64le.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Development on Power (ppc64le) systems is done outside of vagrant, on the
+# native OS. This script helps setup the dev env on ppc64le Ubuntu.
+#
+# See https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/install.md#building-outside-of-vagrant-
+#
+# NOTE: This script assumes that 
+#	- OS specific apt-sources / repositories are already set appropriately.
+#
+# To get started on a fresh Ubuntu install: 
+#	mkdir -p $HOME/git/src/github.com/hyperledger
+#	cd $HOME/git/src/github.com/hyperledger
+#	git clone https://github.com/hyperledger/fabric.git	
+#	sudo ./fabric/devenv/setupUbuntuOnPPC64el.sh
+#	export GOPATH=$HOME/git
+#	cd $HOME/git/src/github.com/hyperledger/fabric
+#	make dist-clean all
+
+if [ xroot != x$(whoami) ]
+then
+   echo "You must run as root (Hint: sudo su)"
+   exit
+fi
+
+if [ ! -d "$HOME/git/src/github.com/hyperledger/fabric" ]
+then
+    echo "Script fabric code is under $HOME/git/src/github.com/hyperledger/fabric "
+    exit
+fi
+
+#####################################
+# Install pre-requisite OS packages #
+#####################################
+apt-get update
+apt-get -y install software-properties-common curl git sudo wget
+
+#sudo apt-get -y install vi
+
+#####################################
+# Install and setup Docker services #
+#####################################
+# Along with docker.io, aufs-tools also needs to be installed as 'auplink' which is part of aufs-tools package gets invoked during behave tests.
+apt-get -y install docker.io aufs-tools
+
+# Set DOCKER_OPTS and restart Docker daemon.
+sed  -i '/#DOCKER_OPTS=/a DOCKER_OPTS="-H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock"' /etc/default/docker
+systemctl restart docker
+
+####################################
+# Install Go and set env variable  #
+####################################
+# Golang binaries for ppc64le are publicly available from Unicamp and is recommended as it includes certain platform specific tuning/optimization.
+# Alternativley package part of Ubuntu disto repo can also be used.
+wget ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/ubuntu/dists/trusty/at9.0/binary-ppc64el/advance-toolchain-at9.0-golang_9.0-3_ppc64el.deb
+dpkg -i advance-toolchain-at9.0-golang_9.0-3_ppc64el.deb
+rm -f advance-toolchain-at9.0-golang_9.0-3_ppc64el.deb
+
+# Create links under /usr/bin using update-alternatives
+update-alternatives --install /usr/bin/go go /usr/local/go/bin/go 9
+update-alternatives --install /usr/bin/gofmt gofmt /usr/local/go/bin/gofmt 9
+
+# Set the GOROOT env variable
+export GOROOT="/usr/local/go"
+
+####################################
```

Comment:
I am concerned about the proliferation of these build instructions being pasted over and over.  This isn't to pick on you specifically @SriramaSharma since others have set the precedent.  However, I do wonder if we should address this now rather than allow the cruft to continue to build up.